sentences/de/_common.yaml

```diff
@@ -573,15 +574,15 @@ expansion_rules:
   luefter: "([(der|den) ]Ventilator|[die ]Ventilatoren|[(der|die|den) ]Lüfter)"
   alle_luefter: "<alle> (Ventilatoren|Lüfter)"
   brightness: "{brightness}[ (Prozent|%)]"
-  temperature: "{temperature}[ ][°|Grad]"
+  temperature: "{temperature} [°|Grad]"
   licht: "[(das|die) ](Licht|Lampe|Beleuchtung)"
   lichtes: "([des ](Lichts|Lichtes)|[der ](Lampe|Beleuchtung))"
   lichter: "[(die|der|von den) ](Lichter|Lichtern|Lampen|Leuchten|Beleuchtungen)"
   alle_lichter: "(<alle> (<lichter>|Beleuchtung)|von allen[ (Lichtern|Lampen|Leuchten|Beleuchtungen)])"
   leuchten_lassen: "([er]leuchten lassen|[ein]färben)"
   hier: "(hier[ (im|in diesem) (Raum|Bereich)]|[(im|in diesem|diesen|den) ](Raum|Bereich))"
-  tuer: "[die ]Tür[e|en]"
-  schloss: "([das ]Schloss|[die ]Schlösser)"
+  tuer: "[die] Tür[e|en]"
+  schloss: "([das] Schloss|[die] Schlösser)"
```

Comment:
What's the reasoning behind that change?

Reply:
since some time ago, spaces in optional words are not needed (as per how it is processing the sentences), so for easy reading i just enclosed the optional word without any additional spaces.

Reply:
Let's progress with this PR as I really dig the benefits. I just don't want to revert something that was intentionally introduced and apart from readability doesn't add value.

Could you remove those space changes from this PR? I think this should be discussed in a separate PR and should include all occurrences of spaces.